### PR TITLE
Added support for Django 1.11

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: nuit/templatetags/nuit.py [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+from distutils.version import StrictVersion
+from django import template, get_version
 from django.contrib.messages import constants
 from django.core.urlresolvers import resolve, reverse, NoReverseMatch, Resolver404
 from django.template.loader import get_template
 from django.template.base import token_kwargs, FilterExpression
 from django.template.loader_tags import do_extends, ExtendsNode
 from django.template.defaultfilters import slugify
 from django.utils.html import format_html
-from ast import literal_eval
```
Comment (pwilczynskiclearcode) [inline nuit/templatetags/nuit.py]: `literal_eval` isn't used anywhere

File: nuit/templatetags/nuit.py [hunk 2]
```diff
@@ -394,10 +399,15 @@ def render(self, context):
         # Each element in a row list is a FoundationFormField object.
 
         try:
-            return form_template.render(context)
+            if IS_DJANGO_1_11_OR_ABOVE:
+                context_dict = context.flatten()
+                return form_template.render(context_dict, context_dict.get('request'))
+            else:
+                return form_template.render(context)
```
Comment (pwilczynskiclearcode) [inline nuit/templatetags/nuit.py]: https://docs.djangoproject.com/en/1.11/releases/1.11/#django-template-backends-django-template-render-prohibits-non-dict-context

File: .travis.yml [hunk 3]
```diff
@@ -28,5 +29,5 @@ deploy:
   distributions: "bdist_wheel sdist"
   on:
     all_branches: true
-    condition: $DJANGO_VERSION = ">=1.10, < 1.11"
+    condition: $DJANGO_VERSION = ">=1.10, < 2.0"
```
Comment (mikebryant) [inline .travis.yml]: This is wrong - it needs to match one of the above entries.

e.g. here it should be `condition: $DJANGO_VERSION = ">=1.11, < 2.0"`

**Review by mikebryant — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (pwilczynskiclearcode) [reply .travis.yml]: Fixed that

**Review by mikebryant — APPROVED** [review]

**Pull request merged**